pandas/tests/indexes/interval/test_interval_new.py

```diff
@@ -21,45 +21,25 @@ def _compare_tuple_of_numpy_array(self, result, expected):
 
     @pytest.mark.parametrize("idx_side", ['right', 'left', 'both', 'neither'])
     @pytest.mark.parametrize("side", ['right', 'left', 'both', 'neither'])
-    def test_get_loc_interval(self, idx_side, side):
-
+    @pytest.mark.parametrize("bound", [
+        [0, 1], [1, 2], [2, 3], [3, 4], [0, 2], [2.5, 3], [-1, 4]], ids=str)
```

Comment:
the entire test was within a `for` loop, pulled it out of the test by using `parametrize`